co_simulation/validation/fsi_turek_FSI2/source/ProjectParametersCoSim.json

```diff
@@ -0,0 +1,114 @@
+{
+    "problem_data" :
+    {
+        "start_time" : 0.0,
+        "end_time" : 20.0,
+        "echo_level" : 0,
+        "print_colors" : true,
+        "parallel_type" : "OpenMP"
+    },
+    "solver_settings" :
+    {
+        "type" : "coupled_solvers.gauss_seidel_strong",
+        "echo_level" : 1,
+        "num_coupling_iterations" : 20,
+        "convergence_accelerators" : [
+            {
+                "type"      : "mvqn",
+                "solver"    : "fluid",
+                "data_name" : "disp"
+            }
+        ],
+        "convergence_criteria" : [
+            {
+                "type"          : "relative_norm_previous_residual",
+                "solver"        : "fluid",
+                "data_name"     : "disp",
+                "abs_tolerance" : 1e-6,
+                "rel_tolerance" : 1e-6
+            }
+        ],
+        "data_transfer_operators" : {
+            "mapper" : {
+                "type" : "kratos_mapping",
+                "mapper_settings" : {
+                    "mapper_type" : "nearest_element"
+                }
+            }
+        },
+        "coupling_sequence":
+        [
+            {
+                "name": "fluid",
+                "input_data_list"  : [],
+                "output_data_list" : []
+            },
+            {
+                "name": "structure",
+                "input_data_list": [
+                    {
+                        "data"             : "load",
+                        "from_solver"      : "fluid",
+                        "from_solver_data" : "load",
+                        "data_transfer_operator" : "mapper",
+                        "data_transfer_operator_options" : ["swap_sign","use_transpose"]
+                    }
+                ],
+                "output_data_list": [
+                    {
+                        "data"           : "disp",
+                        "to_solver"      : "fluid",
+                        "to_solver_data" : "disp",
+                        "data_transfer_operator" : "mapper"
+                    }
+                ]
+            }
+        ],
+        "solvers" :
+        {
+            "fluid":
+            {
+                "type" : "solver_wrappers.kratos.fluid_dynamics_wrapper",
+                "solver_wrapper_settings" : {
+                    "input_file"  : "ProjectParametersCFD"
+                },
+                "data" : {
+                    "disp" : {
+                        "model_part_name"   : "FluidModelPart.FluidNoSlipInterface2D_FluidInterface",
+                        "dimension" : 2,
+                        "variable_name" : "MESH_DISPLACEMENT"
+                    },
+                    "load" : {
+                        "model_part_name"   : "FluidModelPart.FluidNoSlipInterface2D_FluidInterface",
+                        "dimension" : 2,
+                        "variable_name" : "REACTION"
+                    },
+                    "velocity" : {
```

Comment:
I think you don't use velocity in this example. Can you delete it from settings?